src/cdk/drag-drop/drag-ref.ts

```diff
@@ -1044,10 +1044,27 @@ export class DragRef<T = any> {
         // we can get away with it. See https://bugzilla.mozilla.org/show_bug.cgi?id=1615824.
         (event.touches[0] || event.changedTouches[0] || {pageX: 0, pageY: 0}) : event;
 
-    return {
-      x: point.pageX - scrollPosition.left,
-      y: point.pageY - scrollPosition.top
+    let point = {
+      x: eventPoint.pageX - scrollPosition.left,
+      y: eventPoint.pageY - scrollPosition.top
     };
+
+    // if dragging SVG element, try to convert from the screen coordinate system to the SVG
+    // coordinate system
+    if (typeof SVGElement !== 'undefined' && this._rootElement instanceof SVGElement) {
+          const svg = this._rootElement.ownerSVGElement;
```

Comment:
This will only change if the root element changes, so maybe we can pre-compute it inside `withRootElement` and save it in a property?